hyperon_das/das.py

```diff
@@ -21,49 +21,52 @@
 class DistributedAtomSpace:
     def __init__(self, system_parameters: Dict[str, Any] = {}, **kwargs) -> None:
         self.system_parameters = system_parameters
-        self._set_default_system_parameters()
-        atomdb = kwargs.get('atomdb', 'ram')
-        query_engine = kwargs.get('query_engine', 'local')
+        self.atomdb = kwargs.get('atomdb', 'ram')
+        self.query_engine = kwargs.get('query_engine', 'local')
+        self.__set_default_system_parameters()
```

Comment:
Yes, There is a implicit meaning. In Python, double underscore ('__') triggers `name mangling`, where the interpreter changes the name of the variable or method to include the class name. We use this to avoid naming conflicts in subclasses or when we need to ensure that a method or variable is not accidentally overridden in subclass instances.

```
class DistributedAtomSpace:
    def __init__(self, **kwargs) -> None:
        self.__name = 'das'
        
das = DistributedAtomSpace()
das.__name -> Throws an Error

To access this attribute directly you need this:
das._DistributedAtomSpace__name
```

but in this case it is unnecessary I'll remove them